src/QuickMarcEditor/QuickMarcEditWrapper.js

```diff
@@ -132,7 +153,22 @@ const QuickMarcEditWrapper = ({
 
         setHttpError(parsedError);
       });
-  }, [showCallout, refreshPageData, location, initialValues, instance, locations, marcType, mutator]);
+  }, [showCallout, refreshPageData, location, initialValues, instance, locations, marcType, mutator, numOfLinks]);
+
+  useEffect(() => {
+    if (marcType !== MARC_TYPES.AUTHORITY) {
+      return;
+    }
+
+    mutator.quickMarcInstanceLinks.POST({
```

Comment:
It should be invoked only for an authorized record.

Reply:
const authRefType = new URLSearchParams(location.search).get('authRefType');

    if (marcType === MARC_TYPES.AUTHORITY && authRefType === 'Authorized')